resources/js/components/Teams/TeamMap.vue

```diff
@@ -71,6 +79,22 @@ export default {
                 ''
             );
         },
+
+        /**
+         *
+         */
+        fullscreenChange (fullscreen)
+        {
+            this.fullscreen = fullscreen
+        },
+
+        /**
+         *
+         */
+        toggle ()
+        {
+            this.$refs['fullscreen'].toggle() // recommended
+        },
```

Comment:
Comments missing or delete the comment blocks

Reply:
Deleting the comments.